README.md

```diff
@@ -1,29 +1,33 @@
 # SharedSnowplowConsumer
 
 shared consumer for consuming from event-bridge and emitting to snowplow.
+![Architecture](snowplow.png)
+
 Architecutre diagram: https://miro.com/app/board/uXjVO5oHq_U=/
 
 ## Folder structure
 - the infrastructure code is present in `.aws`
-  - to add a new event, add the sns subscription of the event to the `sqsConsumerLambda`
-  - and append the SNS permissions with the existing inline access policy
-  
-
-- the `lambda` folder contains the SQSLambda code. This listens to the SNS and forward 
-  the event and its payload to the ecs application to the `sendEvents` POST endpoints
-  - if the ECS is down or any error thrown by sendEvents endpoint , it will forward 
-    the messages to its DLQ.
-
+  - the SQS should be able to listen to the SNS/eventBridge rule that you want to onboard
 
+  
 - the ECS application code is in `src`. This sends the event to snowplow.
+  - the `sqsConsumer` contains logic to consume from the SQS queue
   - the `eventConsumer` contains logic to consume from event-bridge and transform them to snowplow
   - the `snowplow` folder contains handlers to send events to snowplow
 - `.docker` contains local setup
 - `.circleci` contains circleCI setup
 
+## How to onboard a new event?
+- in `.aws`, add the SNS arn to `SNSTopicsSubscriptionList`
+- in `eventConsumer`, please map the event-bridge payload to your snowplow format.
+- in `snowplow` folder, please construct your snowplow payload as per your schema model
+  - Make sure the schema version in `config.schema` that matches with dev/prod snowplow version
+- Note: please check with #support-data-analytics if the model can tolerate duplicate events before onboarding events to this repo
```

Comment:
Is there a requirement that all events are idempotent? If so I think that should be documented more clearly.

Reply:
yes - all events going through event-bridge need to be idempotent..can add it to the docs